### PR TITLE
refactor(mono): Prepare patch 3

### PR DESCRIPTION
**Pull request by chefilip**

### CODE REVIEW

File: packages/wagmi/package.json [hunk 1]
```diff
@@ -3,9 +3,7 @@
   "version": "0.0.1",
   "license": "WAGMIT",
   "description": "wagmi adaptor for pancakeswap",
-  "main": "./dist/index.js",
```
Comment (chef-jojo) [inline packages/wagmi/package.json:6]: It's a published package

**Review by chef-jojo — CHANGES_REQUESTED** [review]

File: packages/wagmi/package.json (hunk 1 above)
Comment (chefilip) [reply packages/wagmi/package.json:6]: Oh. I see. Revert it.

Comment (chef-jojo) [reply packages/wagmi/package.json:6]: revert all in this file?

Comment (chefilip) [reply packages/wagmi/package.json]: Done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**